core/trino-main/src/main/java/io/trino/server/ui/OAuthIdTokenCookie.java

```diff
@@ -17,48 +17,29 @@
 import jakarta.ws.rs.core.NewCookie;
```

Comment:
Can we apply the same to `Nonce` cookie and authentication cookie which is a part of `FormWebUiAuthenticator` - I'm not sure if they would cross the limit

Reply:
Nonce will be shorter so it doesn't matter.

Reply:
UI cookie added